staging/src/k8s.io/noderesourcetopology-api/manifests/crd.yaml

```diff
@@ -0,0 +1,55 @@
+---
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    "api-approved.kubernetes.io": "https://github.com/kubernetes/enhancements/pull/1870"
+  name: noderesourcetopologies.topology.node.k8s.io
+spec:
+  group: topology.node.k8s.io
+  version: v1alpha1
+  versions:
+    - name: v1alpha1
+      served: true
+      storage: true
+  scope: Namespaced
+  validation:
+    openAPIV3Schema:
+      type: object
+      properties:
+        topologyPolicies:
+          type: array
+          items:
+            type: string
+        zones:
+          type: object
+          additionalProperties:
+            type: object
+            properties:
+              type:
+                type: string
+              parent:
+                type: string
+              resources:
+                type: object
+                additionalProperties:
+                  type: object
+                  properties:
+                    capacity:
+                      x-kubernetes-int-or-string: true
```

Comment:
Is this right?  I am not sure how CRD represents "quantity" which is NOT IntOrString under the covers.

Reply:
I assume quantity could be represented just by string, since there is a method to parse a string. When we discussed it with https://github.com/intel/cri-resource-manager authors, if I'm not wrong @marquiz suggested such new for k8s type. It looked like advantage from end user point of view.

Reply:
Isn't the idea that one can define capacity like "64Gi" and not just raw integers(?)

Reply:
Sorry I missed that before. We don't know how to represent a quantity, and that's a problem we'd like to solve.

Reply:
We stuck to `x-kubernetes-int-or-string` as we didn't manage to find a better alternative. Suggestions are welcome on this.